.rubocop.yml

```diff
@@ -11,15 +11,9 @@ Layout/AlignHash:
 Layout/EmptyLines:
   Enabled: false
 
-Layout/EmptyLinesAroundClassBody:
-  Enabled: false
-
 Layout/EmptyLinesAroundMethodBody:
   Enabled: false
 
-Layout/EmptyLineBetweenDefs:
```

Comment:
Rubocop insists on too few lines between method declarations.  I like to have 1 blank line separating grouped control blocks, no blank lines between a method's doc text and the method signature, and 2-3 lines between method declarations.  If everything is spaced evenly, then it becomes very difficult to pick method declarations out as I'm quickly scanning through text.

Reply:
Again, I prefer consistency (at my last job, my personal preference was 2 lines, but it seemed less important than not surprising my colleagues). If you feel very strongly about it, I don't see a problem with configuring a minimum of 1 and maximum of 2 lines.

Reply:
A minimum of 1 would force us to leave space between the method doc and the method signature, which is hard to scan and looks like the method docstring is just some random comment floating about.

A maximum of 2 would not allow me to leave 3 lines, which I prefer and do consistently, especially in codebases with long methods.

Reply:
We will have a standard of 2 blank lines for this codebase.